bindings/bind_py_models.cc

```diff
@@ -67,7 +121,7 @@ namespace rascal {
    *                   needed but not useful to use on the python side
    *
    */
-  void add_kernels(py::module & mod, py::module & m_internal) {
+  void add_models(py::module & mod, py::module & m_internal) {
```

Comment:
I renamed the function because it effectivelly binds more than just kernel function and to me it's fair to bundle all these functionalities under the model umbrella but its true that we have not discuss this in details already.

Reply:
I see. I made an entry in the meeting minutes.